marketplace/charts/spinkube-azure-marketplace/templates/kwasm-annotate-nodes-job.yaml

```diff
@@ -3,7 +3,7 @@ kind: Job
 metadata:
   name: "{{ .Release.Name }}-kwasm-annotate-nodes"
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
It seems fine to re-labeling nodes for both the post-install and post-upgrade scenarios. Have you tested this? I am asking because I am concerend about if we need to have buffer time of labeling kwasm-node from false to true.

Reply:
I have tested locally, though only on KinD.  Maybe worth checking on AKS?

Oh, I could also add test coverage for this in our helm chart smoke test (or create a new workflow with scenarios like this)...

Reply:
I like the idea of adding more tests